shell/platform/embedder/tests/embedder_test_vulkan.cc

```diff
@@ -0,0 +1,17 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/embedder/tests/embedder_test.h"
+
+#include "flutter/shell/platform/embedder/tests/embedder_test_context_vulkan.h"
+
+namespace flutter {
```

Comment:
From go/fluttercpp: "Single-line nested namespace declarations are preferred in new code, but are not required.". `namespace flutter::testing` instead? Slightly prefer this over the new old way but your call.

Reply:
TIL - will do.